src/scss/components/choice-tile/_index.scss

```diff
@@ -0,0 +1,254 @@
+@import "components/box/config";
+@import "components/grid/config";
+@import "mixins";
+
+hx-tile {
+  @include box-size(md);
+
+  border-radius: 2px;
+  display: inline-block;
+  height: 100%;
+  position: relative;
+  text-align: center;
+  width: 100%;
+
+  > header {
+    font-size: 1rem;
+    font-weight: 700;
+    margin-bottom: 0;
+  }
+
+  > p {
+    font-size: 0.75rem;
+  }
+
+  &.hxSm {
+    $pad: 0.75rem;
+
+    padding: $pad;
+
+    > p {
+      display: none;
+    }
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  &.hxLg {
+    $pad: 1.75rem;
+
+    padding: $pad;
+
+    hx-icon[type="checkmark"] {
+      right: $pad;
+      top: $pad;
+    }
+  }
+
+  > header + *,
+  .hx-tile-icon + * {
+    margin-top: 0.5rem;
+  }
+
+  .hx-tile-icon {
+    hx-icon {
+      font-size: 1.75rem;
+    }
+  }
+
+  hx-icon[type="checkmark"] {
+    display: none;
+    font-size: 1rem;
+    right: 1.25rem;
+    top: 1.25rem;
+  }
+
+  &.hxSubdued {
+    opacity: 0.7;
+
+    &:hover {
+      opacity: 1;
+    }
+  }
+}//hx-tile
+
+.hxRow > label.hxChoice {
+  @include grid-column;
+}
+
+// Layout & Geometry
+// ========== COMMON ==========
+label.hxChoice {
+  position: relative;
+}
+
+label.hxChoice > input {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  border: 0;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  z-index: -10;
+
+  &:focus {
+    @include reset-focus;
+  }
+
+  &:checked + hx-tile {
+    hx-icon[type="checkmark"] {
+      display: block;
+      position: absolute;
+    }
+  }
+}
+
+// ========== SUBDUED ==========
+label.hxChoice > input:focus + hx-tile.hxSubdued,
+label.hxChoice > input + hx-tile:hover.hxSubdued,
+label.hxChoice > input[disabled] + hx-tile.hxSubdued,
+label.hxChoice > input:checked + hx-tile.hxSubdued,
+label.hxChoice > input[invalid] + hx-tile.hxSubdued {
+  opacity: 1;
+}
+
+// ========== DEFAULT ==========
+// NOTE: These styles may be able to go into default hx-tile defs
+label.hxChoice > input + hx-tile {
+  background-color: $gray-0;
+  border: solid 1px $gray-400;
+  color: $gray-900;
+  cursor: pointer;
+
+  .hx-tile-icon {
+    color: $gray-700;
+  }
+}
+
+label.hxChoice > input + hx-tile:hover {
+  background-color: $cyan-50;
+  border-color: $cyan-500;
+}
+// same as hover, with focus glow
+label.hxChoice > input:focus + hx-tile {
+  box-shadow: $focus-glow;
+}
+
```

Comment:
Please remove extra blank line.

Reply:
@100stacks, if the linter isn't complaining, I'd rather let this one go.

Reply:
@CITguy, all of our CSS contains a blank line between declaration blocks.  I prefer to stick with our pattern by adding a blank line here.

If you want to add a rule the linter, please do.